src/components/VideoDetailsSidebar/VideoDetailsSidebar.spec.jsx

```diff
@@ -3,156 +3,7 @@ import { render } from '@testing-library/react';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 import VideoDetailsSidebar from './VideoDetailsSidebar.component';
-
-const mockData = {
-  kind: 'youtube#searchListResponse',
-  etag: 'LRviZfd_p3HDDD2uBk5Qv7zaEQU',
```

Comment:
This mocked data is used in other tests, you can refactor this and eliminate the duplicated code